pkg/build/builder/docker_test.go

```diff
@@ -319,6 +320,63 @@ func TestEmptySource(t *testing.T) {
 		}
 	}
 }
+
+// We should not be able to try to push to scratch
+func TestDockerfileFromScratch(t *testing.T) {
+	strPointer := new(string)
+	*strPointer = `FROM scratch
+USER 1001`
```

Comment:
dockerfile:="FROM scratch" ?
(and then just reference it as `&dockerfile` in the struct below)